Recipes/Mains-Advanced/Surface_Water_Mass_Transformation.ipynb

```diff
@@ -72,7 +72,7 @@
     "warnings.filterwarnings(\"ignore\", category = RuntimeWarning)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    file_storage = "/scratch/x77/jn8053"
</pre><p>let's change this to have:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">import os
</pre><p>above in the imports and then</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># Set temp_dir to a directory to store ~8GB of temporary output files
temp_dir = os.path.expandvars("/scratch/$PROJECT/$USER/temp")
temp_path = pathlib.Path(expanded_path)
</pre><p> </p><p>There may be further changes needed down the road for the new variable names I suggested?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>